catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/helper/MetacardHelper.java

```
@@ -0,0 +1,124 @@
+import java.util.Arrays;
+import java.util.List;
+
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.Marshaller;
+
+import org.apache.commons.io.IOUtils;
+import org.codice.ddf.parser.Parser;
+import org.codice.ddf.parser.ParserConfigurator;
+import org.codice.ddf.parser.ParserException;
+import org.codice.ddf.registry.schemabindings.EbrimConstants;
+
+import com.google.common.base.Charsets;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.AttributeImpl;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryObjectType;
+import oasis.names.tc.ebxml_regrep.xsd.rim._3.RegistryPackageType;
+
+public class MetacardHelper {
+    private Parser parser;
+
+    private ParserConfigurator unmarshalConfigurator;
+
+    private ParserConfigurator marshalConfigurator;
+
+    public MetacardHelper(Parser parser) {
+        setParser(parser);
+    }
+
+    /**
+     * Converts the metacards metadata into a RegistryPackageType object
+     *
+     * @param mcard A metacard with ebrim metadata
+     * @return A RegistryPackageType object created from the ebrim metadata
+     * @throws ParserException
+     */
+    public RegistryPackageType getRegistryPackageFromMetacard(Metacard mcard)
+            throws ParserException {
+        InputStream inputStream = null;
+        JAXBElement<RegistryObjectType> registryObjectTypeJAXBElement = null;
+        try {
+            String metadata = mcard.getMetadata();
+            inputStream = new ByteArrayInputStream(metadata.getBytes(Charsets.UTF_8));
+            registryObjectTypeJAXBElement = parser.unmarshal(unmarshalConfigurator,
+                    JAXBElement.class,
+                    inputStream);
+        } finally {
+            IOUtils.closeQuietly(inputStream);
+        }
+
+        if (registryObjectTypeJAXBElement == null) {
+            throw new ParserException("Error parsing metacards xml as ebrim");
+        }
+        return (RegistryPackageType) registryObjectTypeJAXBElement.getValue();
```

Should we check for a `null` value here and throw an exception instead of returning `null`?

If we need to return `null`, we need to add `@Nullable` and explain when `null` would be returned.